### ISSUE TITLE
Node id lists of length 1 cause an internal server error

### ISSUE THREAD

**Issue by maximusunc** [ranking-agent/strider#82]
```
{
  "message": {
    "query_graph": {
      "nodes": {
        "n0": {
          "id": ["MONDO:0005737"],
          "category": "biolink:Disease"
        },
        "n1": {
          "category": "biolink:Gene"
        }
      },
      "edges": {
        "e01": {
          "subject": "n0",
          "object": "n1"
        }
      }
    }
  }
}
```
currently causes an internal server error, while the following does not:
```
{
  "message": {
    "query_graph": {
      "nodes": {
        "n0": {
          "id": "MONDO:0005737",
          "category": "biolink:Disease"
        },
        "n1": {
          "category": "biolink:Gene"
        }
      },
      "edges": {
        "e01": {
          "subject": "n0",
          "object": "n1"
        }
      }
    }
  }
}
```

**Comment by maximusunc** [comment]
I don't know if a new version was deployed last night or things are just inconsistent, but the list of ids seems to work now.

**Comment by patrickkwang** [comment]
Fixed by #81!
